test_tools/disk_utils.py

```diff
@@ -326,11 +326,13 @@ def _is_by_id_path(path: str):
 
 def _is_dev_path_whitelisted(path: str):
     """check if given path is whitelisted"""
-    if re.search(r"cas\d+-\d+", path) or re.search(r"/dev/dm-\d+", path) is not None:
-        return True
-    else:
-        return False
+    whitelisted_paths = [r"cas\d+-\d+", r"/dev/dm-\d+"]
 
+    for whitelisted_path in whitelisted_paths:
+        if re.search(whitelisted_path, path) is not None:
+            return True
+        else:
+            return False
```

Comment:
This for will return for the first checked value regardless of the result.
You could do:
```suggestion
    for whitelisted_path in whitelisted_paths:
        if re.search(whitelisted_path, path) is not None:
            return True
    return False
```